.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: python
-  python:
-    - 3.6
-    - nightly
-  script:
-    - pytest
+python:
+  - "3.6"
+# command to install dependencies
+install:
+  - pip install .
```

Comment:
Remove this line of code.

Reply:
I meant Line 6. Only do pip install -r requirements.txt if you have a requirements file. Kindly reach out to your team mates on slack for help with this.